casr/src/bin/casr-java.rs

```diff
@@ -56,6 +56,17 @@ fn main() -> Result<()> {
                 .value_name("FILE")
                 .help("File with regular expressions for functions and file paths that should be ignored"),
         )
+        .arg(
+            Arg::new("sub-tool")
+                .long("sub-tool")
+                .default_value("casr-san")
+                .action(ArgAction::Set)
+                .value_parser(clap::value_parser!(PathBuf))
+                .value_name("BIN")
+                .help(
+                    "Path to sub tool bin for crash analyze that will be called in case main tool fails",
```

Comment:
remove `bin`

Reply:
analyze -> analysis

Reply:
called when main tool fails to detect a crash

Reply:
Done